core/network/src/main/java/com/school_of_company/network/util/ExpoApiHandler.kt

```
@@ -0,0 +1,41 @@
+                409 -> ConflictException(message = message)
+                500, 501, 502, 503 -> ServerException(message = message)
+                else -> OtherHttpException(
+                    message = message,
+                    code = e.code()
+                )
+            }
+        } catch (e: SocketTimeoutException) {
+            throw TimeOutException(message = e.message)
+        } catch (e: UnknownHostException) {
+            throw NoInternetException()
+        } catch (e: TokenExpirationException) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)